src/basic/journal-importer.c

```diff
@@ -347,6 +348,16 @@ int journal_importer_process_data(JournalImporter *imp) {
                         /* chomp newline */
                         n--;
 
+                        if (!journal_field_valid(line, sep - line, true)) {
+                                char buf[64], *t;
```

Comment:
so far we never did such arbitrarily sized arrays, but instead used some macro or so. Can we do that here too? i.e. add FIELD_ELLIPSIZE_LIMIT or so?

Reply:
It *is* completely arbitrary unfortunately. But yeah, I should add some define for it.

Reply:
not that this matters too much, but this line still has no comment btw... The one at the top of process_special_field() does however... Given that this already appears twice here I wonder if adding a (local) macro for this might not be worth it after all...